content/en/docs/exposing-a-service/_index.md

```diff
@@ -33,7 +33,7 @@ We're going to do this with the command `{{% param cliToolName %}} expose`:
 ```
 
 {{% onlyWhen openshift %}}
-You will get the error message reading `Error from server (AlreadyExists): services "example-web-go" already exists` here. This is because the `oc new-app` command you executed during lab 3 already created a service. This is the default behaviour of `oc new-app`  while `oc create deployment` doesn't have this functionality.
+You will get the error message reading `Error from server (AlreadyExists): services "example-web-go" already exists` here. This is because the `oc new-app` command you executed during lab 3 already created a service. This is the default behavior of `oc new-app`  while `oc create deployment` doesn't have this functionality.
```

Comment:
British vs. american english, but you're right, we probably use the american variant more often.